docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc

```diff
@@ -1601,9 +1640,9 @@ public class CodeFlowAuthorizationTest {
 === Using KeycloakTestResourceLifecycleManager
 
 Use `KeycloakTestResourceLifecycleManager` for your tests only if there is a good reason not to use `Dev Services for Keycloak`.
-If you need to do the integration testing against Keycloak then you are encouraged to do it with <<integration-testing-keycloak-devservices,Dev services for Keycloak>>.
+If you need to do the integration testing against Keycloak then you are encouraged to do it with <<integration-testing-keycloak-devservices,Dev Services for Keycloak>>.
 
-Start with adding the following dependency:
+Start by adding the following dependency:
```

Comment:
The bearer token guide uses `Start with`, `Start by` is fine if it is better, worth making it consistent IMHO across both docs

Reply:
We modified the tutorial guide slightly to say "First, add the following dependency:" I will make that same update here for consistency.